tests/README.md

```diff
@@ -237,7 +233,7 @@ When running locally, the tests create a port-forward from your NGF Pod to local
 test framework. Traffic is sent over this port. If running on a GCP VM targeting a GKE cluster, the tests will create an
 internal LoadBalancer service which will receive the test traffic.
 
-**Important**: NFR tests can only be run on a GKE cluster.
+**Important**: Functional tests can only be run on a `kind` cluster. NFR tests can only be run on a GKE cluster.
```

Comment:
Why only on kind?

Reply:
Since we are loading the image onto kind as part of the setup. Don't want to have to push the image to GKE, and I don't think these tests really even need to run there anyway.